### ISSUE TITLE
Unable to run migrations, "VERBOSE=1 vendor/bin/oe-eshop-facts" outputs empty path lines

### ISSUE THREAD

**Issue by cesnauskast** [OXID-eSales/oxvm_eshop#32]
Pre-conditions:
OS: Windows 10 Pro
VirtualBox 5.1.6
Vagrant 1.8.7

I've been trying to install 6.0.0 beta1 as described in readme here: https://github.com/OXID-eSales/oxvm_eshop. All described steps have been executed via PHPStorm terminal (Git), but unfortunately I cannot get shop running.
I did some checks and it looks like that scripts which lie within /oxideshop/vendor/bin/ directory can't be executed. After checking what outputs one of the scripts - "vendor/bin/oe-eshop-facts", I've got empty path lines listed.
Any ideas how to solve it?

**Comment by rezonanc-oxid** [comment]
Root cause of this - Virtualbox does not handle symlinks if you're running with regular user privileges.

Just to test the theory I've used a VM of Windows 10 and the only thing I've did was to execute `vagrant up` using administrative privileges which causes Virtualbox to be run with administrative privileges as well.

<img width="1018" alt="windows_10" src="https://cloud.githubusercontent.com/assets/8058763/21096320/2792bce6-c068-11e6-9a6a-42449fd4e9e2.png">

Hope this solves the problem for you as well!
